pom.xml

```diff
@@ -588,6 +558,19 @@
         </pluginManagement>
         <plugins>
             <plugin>
+                <groupId>org.codehaus.gmavenplus</groupId>
+                <artifactId>gmavenplus-plugin</artifactId>
+                <version>1.4</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>compile</goal>
```

Comment:
Ah I just read up on this, keeping both these goals allows `invokedynamic. Looks good.